.pre-commit-config.yaml

```diff
@@ -5,7 +5,7 @@ repos:
       - id: flake8
         args: ["--exclude=*/client/inference_pb2.py,*/client/inference_pb2_grpc.py"]
   - repo: https://github.com/PyCQA/isort
-    rev: 5.10.1
+    rev: 5.11.5
```

Comment:
why do we change the isort version?

Reply:
I test some models such as resnet18/mobilenetv2/shufflenetv2 in classification task and compare mmlab bench, vacc-fp16 and vacc-int8. next version we will test more models and outout the benchmark result.
Here change the isort version because run `pre-commit` should change isore-version with python3.7